src/lib/selectors.ts

```diff
@@ -4,6 +4,11 @@ import type {
   UserUpdateParams,
 } from '@sendbird/chat';
 
+import { FailedMessageHandler, MessageHandler, UserMessage, UserMessageCreateParams } from '@sendbird/chat/message';
+import { GroupChannel, GroupChannelCreateParams, SendbirdGroupChat } from '@sendbird/chat/groupChannel';
+import { OpenChannel, OpenChannelCreateParams, SendbirdOpenChat } from '@sendbird/chat/openChannel';
+import { FileMessage, FileMessageCreateParams, SendableMessage, UserMessageUpdateParams } from '@sendbird/chat/lib/__definition';
```

Comment:
Need to check if we can import few of these from the path `@sendbird/chat/message`